apps/aetx/src/aetx.erl

```diff
@@ -28,6 +28,8 @@
 -export([tx/1]).
 -endif.
 
+-define(MAXINT, 16#FFFFffffFFFFffff).
```

Comment:
It looks this MAXINT is the internal representation of ttl meant to never expire.
* Why not using an atom? (I do not see why this must be int.)
* How is that this is "the max"? How is range defined? It looks to me that [serialized tx may have larger TTL as unbound](https://github.com/aeternity/protocol/blob/d89aea62813f9a39d946f440b2c640838799afdf/serializations.md#dynamic-size-object-serialization).

Reply:
This is the max block height we can have but if one wants to put an even larger number in there it is certainly possible...